specification/frontdoor/resource-manager/readme.typescript.md

```diff
@@ -0,0 +1,12 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+``` yaml $(typescript)
+typescript:
+  azure-arm: true
+  package-name: "@azure/arm-frontdoor"
+  output-folder: "$(typescript-sdks-folder)/packages/@azure/arm-frontdoor"
```

Comment:
Double check the properties in the readme.nodejs.md file for this one. I think they specified a custom client name that should be copied to this readme.typescript.md file.

Reply:
Nothing in the node file: https://github.com/Azure/azure-rest-api-specs/blob/c2785feb94e767034e37c9f61bd769cf81c549de/specification/frontdoor/resource-manager/readme.nodejs.md